packages/sites/ortho-site/webapp/wdkCustomization/js/client/records/Sequences.tsx

```diff
@@ -404,7 +408,8 @@ export function RecordTable_Sequences(
   if (
     mesaRows != null &&
     sortedRows != null &&
-    mesaRows.length !== sortedRows.length
+    (mesaRows.length !== sortedRows.length ||
+      mesaRows.length !== leaves?.length)
```

Comment:
Are both comparisons necessary?

Reply:
Yes, I think so.

`sortRows()` returns the protein table rows in tree leaf order. Proteins that are missing from the tree are an obvious problem. But the tree can also be bigger (more leaves) than the protein table (though Rich says it won't happen) - it's best to play on the safe side.

Just using `mesaRows.length !== leaves?.length` won't protect against a protein table of X rows and a tree with X leaves, but where none of the proteins/leaves match each other.